apps/SageAccountWeb/src/LoginPage.tsx

```diff
@@ -32,16 +32,16 @@ const LoginPage: React.FunctionComponent<LoginPageProps> = ({
               <SourceAppLogo />
             </div>
             <StandaloneLoginForm
-              sessionCallback={() => setIsSessionEstablished(true)}
+              sessionCallback={() => {
+                redirectAfterSSO(returnToUrl)
+              }}
```

Comment:
Not sure if the original state variable was used because this is unclear. If so, we could instead clarify the prop name by renaming it to something like `onSessionEstablished`